themes/finna2/templates/ajax/feed-list.phtml

```diff
@@ -1,44 +1,40 @@
 <!-- START of: finna - ajax/feed-list.phtml -->
 <div class="list-feed<?= $this->images ? ' with-image' : ''; ?>">
-<?php if ($this->title || $this->translateTitle): ?><h2 class="feed-header"><?= $this->translateTitle ? $this->transEsc($this->translateTitle) : $this->title; ?></h2><?php endif; ?>
-<?php if (isset($this->description)): ?><p class="additional-description"><?= $this->translate($this->description)?></p><?php endif; ?>
+<?php if ($this->title || $this->translateTitle): ?><h2 class="feed-header"><?= $this->translateTitle ? $this->transEsc($this->translateTitle) : $this->escapeHtml($this->title) ?></h2><?php endif; ?>
+<?php if (isset($this->description)): ?><p class="additional-description"><?= $this->translate($this->description) // HTML, intentionally left unescaped ?></p><?php endif; ?>
   <ul>
     <?php $items = isset($this->visualItems) ? array_splice($this->items, 0, $this->visualItems) : $this->items; ?>
-    <?php foreach($items as $item): ?>
-    <li>
-      <?php if (isset($item['date'])): ?>
-        <span class="date"<?php if (isset($item['dateFull'])):?> title="<?=$item['dateFull']?>"<?php endif; ?>>
-          <?= $item['date']; ?>
-        </span>
-      <?php endif; ?>
-      <?php if (isset($item['link'])):?>
-      <a<?= $this->linkTarget ? ' target="' . $this->escapeHtmlAttr($this->linkTarget) . '"' : '' ?> href="<?= $this->escapeHtmlAttr($item['link']) . ($this->feedUrl ? '?feedUrl=' . $this->escapeHtmlAttr(urlencode($this->feedUrl)) : '')?>"<?=$item['modal'] ? ' data-lightbox="" data-lightbox-title="' . $this->escapeHtmlAttr($item['title']) . '"' : ''?> data-feed="<?=$this->escapeHtmlAttr($this->feedUrl)?>"><?php endif; ?>
-        <?php if (isset($item['image']) && isset($item['image']['url'])): ?>
-          <img src="<?= $item['image']['url']; ?>" alt="" />
+    <?php foreach ($items as $item): ?>
+      <li>
+        <?php if (isset($item['date'])): ?>
+          <span class="date"<?php if (isset($item['dateFull'])):?> title="<?=$this->escapeHtmlAttr($item['dateFull'])?>"<?php endif; ?>></span>
```

Comment:
Tähän on varmaan joku selitys et katotaan onko item['date'] olemassa ja asetetaan kuitenkin item['dateFull']?

Reply:
Sitä dateFull:ia käytetään vain tuossa title-attribuutissa, date:a sitten tuossa alapuolella. Ja on tuossa dateFull:lle oma tarkistus kuitenkin. Tämä on ollut näin jo ennestään, lisäsin vain escapet.